### PR TITLE
Add meeting notes from 2022-05

### PR DESCRIPTION
**Pull request by pradyunsg**

### CODE REVIEW

File: docs/monthly-meeting/2022-05.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+A quarterly meeting for the WG seems like the right cadence.
+
+Other possible names:
+- Docs Steering Council.
+- Docs Advisory Board.
+- Docs Oversight Board.
+
+The current members of the WG will discuss their name.  It's not a one-way decision, so we can work on it.
+
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-05.md:93]: Forgot to remove this.

```suggestion
```

**Review by AA-Turner — APPROVED** [review]

File: docs/monthly-meeting/2022-05.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+- [ ] Anyone can make pull requests to update the devguide.  Carol & Mariatta can review pull requests.
+
+#### Improving documentation PR review
+
+Ned: Concept of Documentation-focused committers? A pool of people who can merge Docs PRs. The Jupyter team set out guidance for when documentation should be changed.
+For now we should focus on substantive changes whilst we have fewer committers focussed on docs.
+
+Adam: can we have a team of people who are "approved" to review docs changes? Improving the reviewing story for docs, so that core devs can use it as a signal that someone trustworthy has given a +1 to the change.
+
+- [ ] Mariatta will open a ticket to give CAM, Pradyun, and Adam triage permissions for CPython.
```
Comment (Mariatta) [inline docs/monthly-meeting/2022-05.md:119]:
```suggestion
- [x] Mariatta will open a ticket to give Ned, CAM, Pradyun, and Adam triage permissions for CPython.
```

Comment (AA-Turner) [reply docs/monthly-meeting/2022-05.md]: Sorry to omit you @nedbat -- I had assumed you were already a triager!

A

**Review by willingc — APPROVED** [review]
Thanks @pradyunsg and reviewers. Merging.

**Pull request merged**